CMakeLists.txt

```diff
@@ -112,7 +112,7 @@ include(SetupCompilationFailureTests)
 
 include_directories(${CMAKE_SOURCE_DIR}/external)
 include_directories(${CMAKE_SOURCE_DIR}/src)
-include_directories(${CMAKE_SOURCE_DIR})
+include_directories(${CMAKE_SOURCE_DIR}/tests/Unit)
```

Comment:
optional, but it would be safer to add the tests include in `SpectreAddTestLibs.cmake` to only test libs. See #2139 for an example of how to do that

Reply:
agreed, I'll do it in the next test cleanup PR...

Reply:
Sounds good :)